modules/payment/payment.service.js

```diff
@@ -21,8 +22,9 @@ class PaymentService {
         }
       });
     const signString = secret + '|' + Object.values(ordered).join('|');
+
     return crypto
-      .createHash(CRYPTO)
+      .createHash('sha1')
```

Comment:
Let's use more secure SHA-256 algorithm

Reply:
for this reason, Fondy requieres sha1 signature coding